.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,20 +12,16 @@
 ### Description of change
 
 <!--
-  Please be clear and concise what the change is intended to do,
-  why this change is needed, and how you've verified that it
-  corrects what you intended.
+  Please describe:
+  - what the change is intended to do
+  - why this change is needed
+  - how you've verified it
+  - any other context that will help reviewers understand the change
 
   In some cases it may be helpful to include the current behavior
   and the new behavior.
-
-  If the change is related to an open issue, you can link it here.
-  If you include `Fixes #0000` (replacing `0000` with the issue number)
-  when this is merged it will automatically mark the issue as fixed and
-  close it.
```

Comment:
Some OSS projects actually require PRs to have a related issue. If none exists, this forces contributors to first create an issue explaining the problem which their PR solves. Maybe that's something we could consider adopting.